src/osdag/gui/ui_template_for_mac.py

```diff
@@ -5,27 +5,41 @@
 # Created by: PyQt5 UI code generator 5.13.0
 #
 # WARNING! All changes made in this file will be lost!\
+import sys
+import os
+
+# Add the src/osdag directory to the Python path
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../../')))
+
+# Now try importing the module
+from osdag.design_type.connection.column_cover_plate import ColumnCoverPlate
+
 
 from PyQt5 import QtCore, QtGui, QtWidgets
 from ..design_report import reportGenerator
-
+from PyQt5.QtWidgets import (
+    QMainWindow, QVBoxLayout, QLabel, 
+    QComboBox, QWidget, QMessageBox
+)
 
 from PyQt5.QtGui import *
 from PyQt5.QtWidgets import *
 from PyQt5.QtCore import *   
 from .ui_tutorial import Ui_Tutorial
 from .ui_aboutosdag import Ui_AboutOsdag
 from .ui_ask_question import Ui_AskQuestion
+from ..texlive.Design_wrapper import init_display as init_display_off_screen
 
-from design_type.connection.column_cover_plate import ColumnCoverPlate
+from osdag.design_type.connection.column_cover_plate import ColumnCoverPlate
```

Comment:
Just use `..` instead of `osdag` for consistency. Same for other places. This might also have effect when we try to run outside of the containing folder.